.github/workflows/ci-aqua-security-trivy-tests.yml

```diff
@@ -1,6 +1,26 @@
 name: ci-aqua-security-trivy-tests
 on:
   push:
+    paths-ignore:
+      - '.vscode/CODEOWNERS'
+      - '.vscode/*.json'
+      - 'compose/**'
+      - 'contrib/**'
+      - 'dgraphtest/**'
+      - 'licenses/**'
+      - 'paper/**'
+      - 'RFC/**'
+      - 'static/**'
+      - 't/**'
```

Comment:
this should be removed - again if we make a change to our test executable, we would like to see tests run against this change

Reply:
Okay. BTW, are we moving to /dgraphtest ? at some point this /t will be removed?

Reply:
We are a long way off from removing `t.go`.  The idea behind introducing `dgraphtest` is to make writing new integration tests easier for developers.  But our existing test suite will remain on t.go, at least for some time.

Reply:
Nice. I was thinking that would be just an append. Something to use sporadically and manually.